src/feedback.js

```diff
@@ -451,8 +504,12 @@ exports.executeFeedback = function(feedback, bank) {
 		if (idCheck || titleCheck || guidCheck) {
 			return { color: feedback.options.fg, bgcolor: feedback.options.bg };
 		}
+
+		if (feedback.options.tally !== undefined && feedback.options.tally !== '' && this.data.mix[mix][type + 'Tally'].includes(input.key)) {
```

Comment:
I'm not sure your return schemes make sense here ... shouldn't there be and the `if` as shown then an `else` or `else if` for normal bg + fg colors?  Makes no sense you return those in the earlier `if`, leading me to believe in most cases this `png64` would never be returned.